src/core/support/string.c

```diff
@@ -174,6 +188,27 @@ char *gpr_strjoin(const char **strs, size_t nstrs, size_t *final_length) {
   return out;
 }
 
+static void do_nothing(void *ignored) {}
+gpr_slice_buffer *gpr_strsplit(const char *str, const char *sep) {
```

Comment:
Of course the naive O(n*k) solution using gpr_slice_str_cmp() is probably good enough given the limited scope of the function, and this is what I'll do tomorrow unless something better comes up. I think it's worth preserving the slice based API.